### PR TITLE
Feat/mute others videos

### PR DESCRIPTION
**Pull request by Lawton**
Adds the ability for user to mute others. Only works for users who connect after you, need to figure out how to get the socketId of those that are already connected

### CODE REVIEW

File: src/video-client.js [hunk 1]
```diff
@@ -22,7 +24,7 @@ export function openVideo(socket, myPeer) {
         call.answer(stream);
 
         // other user video stream (user A video to user B screen)
-        const [video, videoContainer] = createVideoContainer(false)
+        const [video, videoContainer] = createVideoContainer(false, -1)
```
Comment (Lawton) [inline src/video-client.js:27]: This should really pass the socket.id of whoever connect, but I'm not sure how to get that

Comment (angieology) [reply src/video-client.js:27]: Current user's socket is in socket.id

If you need to pass another users socket it would be in the payload of the event

Comment (angieology) [reply src/video-client.js:27]: Oh right the event params aren't in scope.

there is (every connected user) state stored in the peer object in this file too

Comment (Lawton) [reply src/video-client.js:27]: If we could get the peerId from the call and then translate that to the socketID that would be ideal, if we can't get it working we could easily hide it in the demo as well

**Review by angieology — APPROVED** [review]
because of the hardcoded socket id there is an issue for the second joining user, it seems like they can't mute/stop their own video, will stop the first user's video

but the feature is a great demonstration of muting, it would be smarter for us to just merge this

File: src/video-client.js (hunk 1 above)
Comment (angieology) [reply src/video-client.js:27]: i agree this is definitely good enough

Comment (ChrisZhangProdigy) [reply src/video-client.js:27]: A hacky way could be passing the caller Id to server and letting the server pass it to receiver...

**Comment by angieology** [comment]
sorry i caused some merge conflicts although the changes are small, we can fix them together tomorrow

**Comment by angieology** [comment]
sorry i caused some merge conflicts although the changes are small, we can fix them together tomorrow

File: src/video-client.js (hunk 1 above)
Comment (angieology) [reply src/video-client.js:27]: And that's why it's called a hackathon

**Pull request merged**